Components/button.css

```diff
@@ -0,0 +1,217 @@
+.btn-section{
+  margin: 2rem 1rem;;
+}
+
+.simple-btn{
+  display: grid;
+  grid-template-columns: 130px 130px 130px;
+  grid-column-gap: 150px;
+  margin-bottom: 1rem;
+}
+
+.btn{
+  color: #212529;
+  text-decoration: none;
+  cursor: pointer;
+  background-color: transparent;
+  border: 1px solid transparent;
+  padding: .7rem 1rem;
+  margin: 0rem 0.5rem;
+  font-size: 1rem;
+  border-radius: 0.3rem;
+}
+
+
+
+/* Filled Button */
+.filled-btn{
+  display: flex;
+}
+
+.btn-primary-fill{
+  color: #fff;
+  background-color: var(--primary-color);
+}
+
+.btn-secondary-fill{
+  color: black;
+  background-color: var(--secondary-color);
+  border-color: var(--primary-color);
+}
+
+.filled-btn > .btn:hover{
+  opacity: 0.5;
+}
+/******/
+
+
+
+/* Ouline Button */
+.outline-btn{
+  display: flex;
+}
+
+.btn-primary-outline{
+  border-color: var(--primary-color);
+  color: var(--primary-color);
+}
+
+.btn-secondary-outline{
+  border-color: var(--secondary-color);
+}
+
+.outline-btn > .btn-primary-outline:hover{
+  background-color: var(--primary-color);
+  color: white;
+  font-weight: bolder;
+}
+
+.outline-btn > .btn-secondary-outline:hover{
+  background-color: var(--secondary-color);
+}
+/*******/
+
+/* Link button*/
+.link{
+  margin-bottom: 1rem;
+}
+.btn-link{
+  color: var(--dark-grey);
+  text-decoration: none;
+  font-weight: bold;
+  cursor: pointer;
+  background-color: transparent;
+  border: 0px solid transparent;
+  padding: .7rem .2rem;
+  margin: 0rem 0rem;
+  font-size: 1rem;
+  border-radius: 0.3rem;
+}
+/*******/
+
+
+.btn-icon-section{
+  margin: 0rem 0.1rem;
+  margin-bottom: 1rem;
+}
+
+.btn-icon-section > a{
+  text-decoration: none;
+}
+
+.btn-icon{
+  padding: 1rem 1rem;
+  margin: 1rem 1rem;
+} 
+
+.icons{
+  width: 40px;
+  height: 40px;
+  margin-top: 2rem;
+}
+
+.icons:hover{
+  color: var(--secondary-color);
+  padding: 0rem 0rem;
+  background-color: var(--secondary-color);
+  padding: 0.5rem 0.5rem;
+}
+
+.fa-icon{
```

Comment:
It would be great if you let the fa classes only for Font Awesome. It will help differentiate. You can use a span and give class to that for styling icons.
You can change it at all the places.